Modules/Sources/NineAnimatorNativeSources/AnimeSources/ArrayAnime/Arrayanime+Episode.swift

```diff
@@ -57,16 +57,16 @@ extension NASourceArrayanime {
             var episodeSource: String = ""
             
             if link.server == "gstore" {
-                if let index = episodeResponse.links.firstIndex(where: { $0.size.contains("High Speed") }) {
-                    episodeSource = episodeResponse.links[index].src
+                if let index = episodeResponse.links.firstIndex(where: { $0.name.contains("(HDP - mp4)") }) {
+                    episodeSource = episodeResponse.links[index].link
                 }
             } else if link.server == "cloud9" {
```

Comment:
It seems like arrayanime doesn't use Cloud9 Servers anymore but instead uses gogo servers. It seems like the gogo server doesn't seem to work (i get 403 errors). Anyways, i just recommend removing this code and just keep the google server, since we already support GogoAnime. Otherwise it LGTM!